pkg/virtctl/imageupload/imageupload.go

```
@@ -0,0 +1,355 @@
+			return v.run(cmd, args)
+		},
+	}
+	cmd.Flags().BoolVar(&insecure, "insecure", insecure, "Allow insecure server connections when using HTTPS.")
+	cmd.Flags().StringVar(&uploadProxyURL, "uploadproxy-url", "", "The URL of the cdi-upload proxy service.")
+	cmd.MarkFlagRequired("uploadproxy-url")
+	cmd.Flags().StringVar(&pvcName, "pvc-name", "", "The destination PVC.")
+	cmd.MarkFlagRequired("pvc-name")
+	cmd.Flags().StringVar(&pvcSize, "pvc-size", "", "The size of the PVC to create (ex. 10Gi, 500Mi).")
+	cmd.Flags().StringVar(&storageClass, "storage-class", "", "The storage class for the PVC.")
+	cmd.Flags().StringVar(&accessMode, "access-mode", accessMode, "The access mode for the PVC.")
+	cmd.Flags().StringVar(&imagePath, "image-path", "", "Path to the local VM image.")
+	cmd.MarkFlagRequired("image-path")
+	cmd.Flags().BoolVar(&noCreate, "no-create", noCreate, "Don't attempt to create a new PVC.")
+	cmd.Flags().UintVar(&uploadPodWaitSecs, "wait-secs", uploadPodWaitSecs, "Seconds to wait for upload pod to start.")
+	cmd.SetUsageTemplate(templates.UsageTemplate())
+	return cmd
+}
+
+func usage() string {
+	usage := `  # Upload a local disk image to a newly created PersistentVolumeClaim:
+	virtctl image-upload --insecure --upload-proxy-url=https://cdi-uploadproxy.mycluster.com --pvc-name=upload-pvc --pvc-size=10Gi --image-path=/images/fedora28.qcow2`
+	return usage
+}
+
+type command struct {
+	clientConfig clientcmd.ClientConfig
+}
+
+func (c *command) run(cmd *cobra.Command, args []string) error {
+	file, err := os.Open(imagePath)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	namespace, _, err := c.clientConfig.Namespace()
+	if err != nil {
+		return err
+	}
+
+	virtClient, err := kubecli.GetKubevirtClientFromClientConfig(c.clientConfig)
+	if err != nil {
+		return fmt.Errorf("Cannot obtain KubeVirt client: %v", err)
+	}
+
+	pvc, err := getUploadPVC(virtClient, namespace, pvcName, noCreate)
+	if err != nil {
+		return err
+	}
+
+	if pvc == nil {
```

Can you return an error above and use `errors.IsNotExist(err)`?